hoot-core/src/main/cpp/hoot/core/elements/Tags.cpp

```diff
@@ -235,6 +236,29 @@ QStringList Tags::getNames() const
   return result;
 }
 
+QStringList Tags::getNamesRecursive() const
+{
```

Comment:
Dunno. Does this include your change? Seems like it does.


https://github.com/ngageoint/hootenanny/pull/50/files#r44556603?


Jason R. Surratt
303-254-2122 (x32122) office
________________________________
From: Brandon Witham <notifications@github.com>
Sent: Wednesday, November 11, 2015 10:02 AM
To: ngageoint/hootenanny
Cc: Jason Surratt
Subject: Re: [hootenanny] 5722 (#50)


In hoot-core/src/main/cpp/hoot/core/elements/Tags.cpp<https://github.com/ngageoint/hootenanny/pull/50#discussion_r44556603>:

> @@ -235,6 +236,29 @@ QStringList Tags::getNames() const
>    return result;
>  }
>
> +QStringList Tags::getNamesRecursive() const
> +{


Made that change and it looks good with the tests....pushed to 5722 branch. Do I need to create a new pull request now? Or is it possible to make the original one pick up that change?

-
Reply to this email directly or view it on GitHub<https://github.com/ngageoint/hootenanny/pull/50/files#r44556603>.

This electronic communication and any attachments may contain confidential and proprietary information of DigitalGlobe, Inc. If you are not the intended recipient, or an agent or employee responsible for delivering this communication to the intended recipient, or if you have received this communication in error, please do not print, copy, retransmit, disseminate or otherwise use the information. Please indicate to the sender that you have received this communication in error, and delete the copy you received.

DigitalGlobe reserves the right to monitor any electronic communication sent or received by its employees, agents or representatives.